ui/pages/extensions.js

```diff
@@ -105,8 +105,49 @@ const MeshMapSnapShotCard = ({ classes, githubActionEnabled = false }) => {
   );
 };
 
+const ServiceMeshPerformanceCard = ({ classes, githubActionEnabled = false }) => {
+  const handleEnable = (e) => {
+    window.open(
+      'https://github.com/marketplace/actions/performance-testing-with-meshery',
+      '_blank',
+    );
+    e.stopPropagation();
+  };
+
+  return (
+    <Grid item {...LARGE_6_MED_12_GRID_STYLE}>
+      <div className={classes.card}>
+        <Typography className={classes.frontContent} variant="h5" component="div">
+          Service Mesh Performance
+        </Typography>
+
+        <Typography className={classes.frontSideDescription} variant="body">
+          <img className={classes.img} src="/static/img/meshmap.svg" />
+          This GitHub Action automatically tests your services performance by simulating user load.
+          It measures things like response time and tracks changes over time.By integrating it into
```

Comment:
Thanks for this PR, @kanugurajesh! There's missing whitespace between the two sentences here.

Reply:
@leecalcote I have resolved the issue